composer.json

```diff
@@ -12,11 +12,20 @@
     "require": {
     },
     "require-dev": {
-        "thefrozenfire/swagger-client": "^2.0"
+        "thefrozenfire/swagger-client": "^2.0",
+        "phpunit/phpunit": "~4.8.28"
     },
     "autoload": {
         "psr-0": {
             "CallFire": "src/"
-        }
+        },
+        "classmap": [
+            "test/"
+        ]
+    },
+    "scripts": {
+        "post-install-cmd": [
```

Comment:
Are we sure we want this to run on install? In certain situations this might cause errors that would break the install process (e.g. with a locked-down corporate firewall preventing swagger.json access).

It might be preferable to simply have the developer run this command manually and maybe include some post-install output to instruct them of this option.

Having a parameter for the command script to point to a custom swagger.json path might also be helpful.